dm_cli/import_package.py

```diff
@@ -341,6 +359,11 @@ def package_tree_from_zip(data_source_id: str, zip_package: io.BytesIO) -> Packa
                 }
             )
 
+        # remove package.json files
+        for file in zip_file.filelist:
+            if file.filename in filenames_to_remove:
+                zip_file.filelist.remove(file)
```

Comment:
Why do you need to remove them from the zipfile?

Reply:
hmm, thought this was necessary to avoid interpreting package.json as a regular entity in the package. But seems like it works without removing from zip_file.

Updated!